features/wildcard.feature

```diff
@@ -0,0 +1,95 @@
+# language: nl
+
+Functionaliteit: zoeken met een wildcard wordt ondersteund op enkele parameters
+  Het is mogelijk te zoeken met een wildcard met de volgende parameters van de ingeschreven persoon:
+  - naam__geslachtsnaam
+  - naam__voornamen
+  - naamopenbareruimte
+
+  Ondersteund worden twee soorten wildcard:
+  - "*": Komt overeen met nul of meer (niet-spatie) karakters
+  - "?": Komt precies overeen met één (niet-spatie) karakter
+
+  Een wildcard kan geplaatst worden aan het begin van de zoekstring en/of aan het eind van de zoekstring.
+
+  De "?" wildcard kan meerdere keren achter elkaar worden opgenomen. Bijvoorbeeld zoeken op "groen??" zoekt op namen die beginnen met "groen" en exact 7 karakters bevatten (bijv. Groenlo en Groenen, maar niet Groen of Groens).
+
+  Bij het zoeken op de parameters die een wildcard ondersteunen, moeten er minimaal 2 karakters exclusief de wildcard(s) worden opgegeven.
+
+  Achtergrond:
+    Gegeven de registratie ingeschreven personen kent zoals beschreven in testdata.csv
+
+  Scenario: Voor attribuut geslachtsnaam wordt zoeken met een wildcard ondersteund
+    Gegeven de registratie ingeschreven personen kent met de geslachtsnaam "Groen", "Groenlo", "Groenink", "Groenen"
```

Comment:
Gegeven de registratie ingeschreven personen kent **personen** met de geslachtsnaam "Groen", "Groenlo", "Groenink", "Groenen" <= volgens mij moet de vetgedrukte woord worden toegevoegd

Reply:
Ik heb de feature 1 op 1 overgenomen van BRP-bevragen. Het is maar hoe je de zin leest. Misschien een keer tijdens een sessie bespreken ?